packages/fast-components/src/checkbox/checkbox.ts

```diff
@@ -38,6 +38,9 @@ export class Checkbox extends FormAssociated<HTMLInputElement> {
         this.defaultChecked = this.checkedAttribute;
     }
 
+    @attr
+    public disabled: boolean;

```

Comment:
Do we need to make this `@attr({ mode: 'boolean' })` or `@attr({mode: 'fromView', converter: booleanConverter })` Ping @chrisdholt for the latest recommendation based on new components.